test/unittest/tests/block/test_block_session_bms.cpp

```diff
@@ -0,0 +1,121 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * [block_session_bms]: block_session.c
```

Comment:
What is the difference between this and test_block_session_ext.cpp? The functions being tested are the same I think.

Reply:
This file tests the block manager creation/deletion and a function that combines both the usage extent and size blocks.